jdbc-mysql/src/main/java/io/kestra/repository/mysql/MysqlRepository.java

```diff
@@ -47,4 +48,12 @@ public <R extends Record, E> ArrayListTotal<E> fetchPage(DSLContext context, Sel
             DSL.using(configuration).fetchOne("SELECT FOUND_ROWS()").into(Integer.class)
         ));
     }
+
+    public Field<Integer> weekFromTimestamp(Field<Timestamp> timestampField) {
```

Comment:
It's not clear to me which is the default and if we cannot just always use mode 3.

Reply:
If we use mode 3 all the time you end up with the first week of some years being counted for the previous year which brings a week computation of > 366 days which crashes